index.js

```diff
@@ -22,83 +22,407 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
-import sh from 'execSync';
+//import sh from 'execSync';
+import i2c from 'i2c-bus';
+import { execSync } from 'child_process';
 import { Peripheral } from 'raspi-peripheral';
+import { VERSION_1_MODEL_B_REV_1, getBoardRevision } from 'raspi-board';
+
+if (typeof execSync !== 'function') {
+  execSync = require('execsync');
+}
+
+function checkAlive(alive) {
```

Comment:
checkAlive will be changed to a method in the class. The rest will remain standalone functions.